shared/modules/snap-accounts.ts

```diff
@@ -0,0 +1,20 @@
+///: BEGIN:ONLY_INCLUDE_IN(build-main)
+/**
+ * Compares the current date to a specific date in UTC.
+ * This method is meant to be used in a specific case related to the
+ * Snap Accounts API. It should not be used in other cases.
+ * Note: Months are 0-indexed in JS, so 10 represents November
+ *
+ * @returns true if the current date is after the specified date, false otherwise.
+ */
+export function showSnapAccountExperimentalToggle(): boolean {
+  // eslint-disable-next-line prefer-destructuring
```

Comment:
I think we should avoid introducing this.

Reply:
https://github.com/MetaMask/metamask-extension/pull/21221#discussion_r1355820024

Reply:
Solved in [0679f5f](https://github.com/MetaMask/metamask-extension/pull/21221/commits/0679f5f531909f2e9cf33a8ec0091963f158ba14)